src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookTests.java

```diff
@@ -73,7 +73,7 @@ public void userCanLoginViaFacebook() {
     dropDown.openWikiPage(wikiURL);
     dropDown.appendToUrl("noads=1");
     dropDown.openDropDown();
-    dropDown.logInViaFacebook(user.getEmail(), user.getPassword());
+//    dropDown.logInViaFacebook(user.getEmail(), user.getPassword());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)